manager/integration/tests/common.py

```diff
@@ -3045,6 +3084,22 @@ def delete_storage_class(sc_name):
         assert e.status == 404
 
 
+def cleanup_storage_class():
+    delete_storage_class(DEFAULT_STORAGECLASS_NAME)
+    ok = False
+    api = get_storage_api_client()
+    for _ in range(RETRY_COUNTS):
+        ok = True
+        ret = api.list_storage_class()
+        for sc in ret.items:
+            if sc.metadata.name == DEFAULT_STORAGECLASS_NAME:
```

Comment:
```
if sc.metadata.name != DEFAULT_STORAGECLASS_NAME:
```

Reply:
Don't think that will work because we only want to delete the *extra* storageClass.

Reply:
Oh sorry. `DEFAULT_STORAGECLASS_NAME` means the default SC of the integration test rather than `longhorn`. my fault.

Reply:
True, it's better to remove everything else. Addressed.